MTA-6811, 6812, 6813, 6814, 6815, 6816 CQA April Developer Lightspeed work (#357)
migtools/mta-documentation: docs/topics/developer-lightspeed/con_llm-service-openshift-ai.adoc

[role="_abstract"]
The code suggestions from {mta-dl-full} differ based on the large language model (LLM) that you use. Therefore, you may want to use an LLM that caters to your specific requirements.

> {mta-dl-plugin} integrates with LLMs that you deployed as a scalable service on {ocp-name} AI clusters. These deployments give you granular control over resources such as compute, cluster nodes, and auto-scaling Graphical Processing Units (GPUs) while enabling you to use LLMs to resolve code issues at a large scale.

Reviewer: Reducing wordiness:
```suggestion
The {mta-dl-plugin} integrates with LLMs deployed as a scalable service on {ocp-name} AI clusters. This integration provides granular control over compute resources, cluster nodes, and auto-scaling Graphical Processing Units (GPUs) to resolve code issues at scale."
```

** Configure an OpenAI API key
** Update the OpenAI API key and the base URL in `provider-settings.yaml`.

> See xref:llm-provider-settings_{context}[Configuring LLM provider settings] to configure the base URL and the LLM API key in the {mta-dl-plugin} Visual Studio Code extension.

Reviewer 2: Per DITA standards, links are not allowed in the Concept body. They must be moved to Additional resources. This is also applicable to assemblies. Fix it if it occurs elsewhere too.
